File: 프로그래머스/unrated/120923. 연속된 수의 합/연속된 수의 합.js
function solution(num, total) {
    const start = parseInt(total/num) - parseInt((num-1)/2);
    const end = parseInt(total/num) + parseInt((num-1)/2);
    const result = [];
    
    let sum = 0;
    for(let i = start; i <= end; i++){
        result.push(i);
        sum += i;
    }
    if(sum < total) result.push(total-sum);
    
    return result;
}

File: 프로그래머스/unrated/120923. 연속된 수의 합/연속된 수의 합.py
def solution(num, total):
    start = (total // num) - ((num - 1) // 2);
    end = (total // num) + ((num - 1) // 2);
    result = [i for i in range(start, end + 1)];
    if sum(result) < total: result.append(total - sum(result));
    return result